bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -146,15 +148,17 @@ public synchronized void login() {
                 logger.debug("SomfyTahoma version: {}", data.getVersion());
                 updateStatus(ThingStatus.ONLINE);
             } else {
-                logger.debug("Login response: {}", response.getContentAsString());
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, "Error logging in");
-                throw new SomfyTahomaException(response.getContentAsString());
+                logger.trace("Login response: {}", response.getContentAsString());
```

Comment:
Because `getContentAsString()` does expensive string operations it makes sense to always conditionally execute it when using it with logging: `if (logger.isTraceEnabled()) {  ... }`